user/enterprise/ssl-certificate-management.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Travis CI Enterprise SSL Certificate Management
+layout: en_enterprise
+---
+
+This page contains information relating to SSL certificates and their use in Travis CI Enterprise.
+
+By default, Travis CI Enterprise verifies SSL for all traffic between its various components. SSL verification helps ensure that traffic is delivered between components securely. It is **strongly recommended** that you configure your installation with a valid certificate from a trusted authority.
```

Comment:
'various component' -> we could be specific here.

Reply:
Addressed in 51e00208.